subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r33/BuildProgressCrossVersionSpec.groovy

```diff
@@ -263,7 +263,6 @@ class BuildProgressCrossVersionSpec extends ToolingApiSpecification {
 
         then:
         !events.operations.find { it.name.contains("compileClasspath") }
-        events.operation("Run tasks").children.empty
```

Comment:
@adammurdoch this might be problem. I can't tell because I'm not sure what this is actually testing.

The underlying change is that we now fire a build operation when finalizing the build cache configuration. This happens the first time someone asks for the `BuildCacheService`. This happens as part of bootstrapping `TaskExecuter`, which happens the first time `Task.execute()` is called.

This doesn't _seem_ like a problem to me.